SharpClipboard/Views/ClipboardHandle.cs

```diff
@@ -227,6 +227,8 @@ protected override void WndProc(ref Message m)
                         }
                     }
 
+                    SendMessage(_chainedWnd, m.Msg, m.WParam, m.LParam);
```

Comment:
Forwards `WM_DRAWCLIPBOARD` to any next instances, providing support for multi-instance clipboard monitoring.